ADOPTERS.md

```diff
@@ -0,0 +1,43 @@
+## End-users
+
+Known end-users include: Bloomberg, Bytedance, Pinterest, Square, and Uber.
+
+SPIFFE and SPIRE are being used by numerous other companies, both large and small, to build higher layer products and services. This includes Decipher Technology Studios, F5 Networks, HashiCorp, Intel, Google, IBM, Tigera, VMware, and many others.
+
+## Ecosystem
+
+SPIFFE and SPIRE have integrations available with a number of open-source projects. The list includes:
+
+Kubernetes
```

Comment:
This needs to be a bullet list or the names all run together on the same line like
Docker Envoy Istio...

So do:

* Docker
* Envoy
* Istio
...